tools/ci_build/github/linux/docker/Dockerfile.manylinux2014_training_cuda11_3

```diff
@@ -1,187 +0,0 @@
-ARG BASEIMAGE=nvcr.io/nvidia/cuda:11.3.1-cudnn8-devel-centos7
```

Comment:
This file is needed by our CI pipeline I believe. I think you can submit another PR for updating the CUDA version in our CI pipeline to 11.6.

Reply:
On reviewing again, this file is not needed as far as I can tell. It is ok to remove.